src/System.Memory/src/System.Memory.csproj

```diff
@@ -112,6 +112,8 @@
     <Compile Include="$(CommonPath)\CoreLib\System\Runtime\InteropServices\MemoryMarshal.cs" />
     <Compile Include="System\ReadOnlySpan.cs" />
     <Compile Include="System\Span.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Span.Shared.cs" />
```

Comment:
The shared implementation should have no suffix. The fast-span specific implementation should have .Fast.cs suffix. It is the convention that we have been using for this so far.

Reply:
ok, will rename the files then. 

{Readonly}Span.cs -> {ReadOnly}Span.Fast.cs
{Readonly}Span.Shared.cs -> {Readonly}Span.cs